wifiphisher/data/phishing-pages/device-modal/config.ini

```diff
@@ -0,0 +1,6 @@
+[info]
+Name: Device Modal
+Description: Shows a login modal based on the victim's device/browser.
```

Comment:
We need more analysis here.

What is your vision regarding this scenario? How would users use it (what credentials can be obtained) and what makes it different from the ones we currently support?